CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * [#6036](https://github.com/rubocop-hq/rubocop/issues/6036): Make `Rails/BulkChangeTable` aware of string table name. ([@wata727][])
 * [#5467](https://github.com/bbatsov/rubocop/issues/5467): Fix a false negative for `Style/MultipleComparison` when multiple comparison is not part of a conditional. ([@koic][])
 * [#6042](https://github.com/rubocop-hq/rubocop/pull/6042): Fix `Lint/RedundantWithObject` error on missing parameter to `each_with_object`. ([@Vasfed][])
+* Support string timestamps in Rails/CreateTableWithTimestamps cop. ([@drn][])
```

Comment:
Since there is no related issue, adding a link to this PR at the beginning is kind to users who sees CHANGELOG.md.

```diff
-* Support string timestamps in Rails/CreateTableWithTimestamps cop. ([@drn][])
+* [#6056](https://github.com/rubocop-hq/rubocop/pull/6056): Support string timestamps in Rails/CreateTableWithTimestamps cop. ([@drn][])
```

Reply:
👍